shell/platform/windows/flutter_window.cc

```diff
@@ -69,16 +69,24 @@ static HCURSOR GetCursorByName(const std::string& cursor_name) {
 }  // namespace
 
 FlutterWindow::FlutterWindow(int width, int height)
-    : binding_handler_delegate_(nullptr) {
+    : binding_handler_delegate_(nullptr), restored_(false), focused_(false) {
```

Comment:
Consider setting these default values on the fields themselves to shorten this line. I don't feel strongly about this

Reply:
Agreed. In fact, bool class fields initialise to false by default, but from a readability point of view, agreed it's better to be explicit about it to show the intent.